.github/workflows/create-package.yml

```diff
@@ -0,0 +1,42 @@
+﻿name: Create and publish package
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  package-release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        configuration: [BelowZero, Subnautica]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get BZ artifact from build
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: build.yml
+        name: SCHIZO-${{ matrix.configuration }}.dll
+        path: Install/${{ matrix.configuration }}/BepInEx/plugins/SCHIZO/SCHIZO.dll.extracted

```

Comment:
Why not set the path to `Install/${{ matrix.configuration }}/BepInEx/plugins/SCHIZO/` and bypass the "Clean extracted artifact" step below

Reply:
Ah, should have properly read the documentation. I thought `path` in this context takes in a file, just like how `upload-artifacts` does, so I tried it [for this run](https://github.com/jameshi16/Neurosama-Subnautica-Mod/actions/runs/6372180818/job/17294756009) and thought it was just doing some weird naming thing.

Thank you